dev.skynomads.Seabird.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: dev.skynomads.Seabird
```

Comment:
skynomads.dev doesn't seem to resolve. If you own it can you put something up?

Reply:
I put a redirect on https://seabird.skynomads.dev/, good enough?

Reply:
Yes, thanks